src/share/cxx/Tracers.hpp

```diff
@@ -28,7 +28,6 @@ struct Tracers {
   ExecViewManaged<Scalar*[Q_NUM_TIME_LEVELS][QSIZE_D][NP][NP][NUM_LEV]> qdp;
   ExecViewManaged<Scalar*[QSIZE_D][NP][NP][NUM_LEV]> qtens_biharmonic; // Also doubles as just qtens.
   ExecViewManaged<Scalar*[QSIZE_D][2][NUM_LEV]> qlim;
-  ExecViewManaged<Scalar*[QSIZE_D][NP][NP][NUM_LEV]> q;
```

Comment:
The fact that we need q for physics-dynamics coupling and not just diagnostics output means we need to keep q here.

Reply:
For physics-dynamic coupling it will be convenient to have NUM_LEV at the end still, right?

Reply:
I suppose in principle. In practice, (i) the coupling subroutine UniquePoints largely makes the issue moot in v1, and (ii) the copy cost is extremely negligible whether considering v1 or NGD physics. We should worry more about (ii), so NUM_LEV is where it should be. For (i), we'll end up having to copy to Fortran elem(.)%state, anyway, until at some point in the future, we streamline (i) at the level of `stepon.F90`.